atom/browser/ui/inspectable_web_contents_impl.cc

```diff
@@ -75,6 +75,9 @@ const char kTitleFormat[] = "Developer Tools - %s";
 
 const size_t kMaxMessageChunkSize = IPC::Channel::kMaximumMessageSize / 4;
 
+// Stores all instances of InspectableWebContentsImpl.
+InspectableWebContentsImpl::List g_web_contents_instances_;
```

Comment:
Not a necessary change for this PR but can lazy construct this with `base::NoDestructor`